public/app/features/alerting/routes.tsx

```diff
@@ -18,7 +18,7 @@ const legacyRoutes: RouteDescriptor[] = [
   {
     path: '/alerting',
     component: () =>
-      config.featureToggles.topnav ? <NavLandingPage navId="alerting" /> : <Redirect to="/alerting/list" />,
+      config.featureToggles.topnav ? <NavLandingPage navId="alerting-legacy" /> : <Redirect to="/alerting/list" />,
```

Comment:
Why do we redirect to legacy alerting?

Reply:
@joshhunt these are the legacy alerting routes. further down in that file there's unified alerting routes. this is just setting the navId correctly so the landing page works properly, otherwise when you click the `Alerting (legacy)` link you get a 404 page 😄